start_cerberus.py

```diff
@@ -44,6 +45,9 @@ def main(cfg):
             kubeconfig_path = None
         logging.info("Initializing client to talk to the Kubernetes cluster")
         kubecli.initialize_clients(kubeconfig_path)
+        sdn_namespace = kubecli.check_sdn_namespace()
```

Comment:
This is going to be an expensive operation when we have thousands of workload generated namespaces in the cluster, maybe parse a config to know about what SDN plugin is in use? Thoughts?